test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeArrayTests.java

```diff
@@ -430,243 +430,243 @@ static class SomeIdentityClassImplIf implements SomeInterface2WithSingleImplemen
    }
 
    static class SomeClassHolder {
-      public static int ARRAY_LENGTH = 10;
-      SomeInterface1WithSingleImplementer[] data_1;
-      SomeInterface1WithSingleImplementer[] data_2;
+		public static int ARRAY_LENGTH = 10;
+		SomeInterface1WithSingleImplementer[] data_1;
+		SomeInterface1WithSingleImplementer[] data_2;
 
-      SomeInterface2WithSingleImplementer[] data_3;
-      SomeInterface2WithSingleImplementer[] data_4;
-      SomeInterface2WithSingleImplementer   data_5;
+		SomeInterface2WithSingleImplementer[] data_3;
+		SomeInterface2WithSingleImplementer[] data_4;
+		SomeInterface2WithSingleImplementer   data_5;
 
 		SomeClassHolder() {
-         data_1 = new SomePrimitiveClassImplIf[ARRAY_LENGTH];
-         data_2 = new SomePrimitiveClassImplIf[ARRAY_LENGTH];
+			data_1 = new SomePrimitiveClassImplIf[ARRAY_LENGTH];
+			data_2 = new SomePrimitiveClassImplIf[ARRAY_LENGTH];
 
-         data_3 = new SomeIdentityClassImplIf[ARRAY_LENGTH];
-         data_4 = new SomeIdentityClassImplIf[ARRAY_LENGTH];
+			data_3 = new SomeIdentityClassImplIf[ARRAY_LENGTH];
+			data_4 = new SomeIdentityClassImplIf[ARRAY_LENGTH];
 
-         data_5 = new SomeIdentityClassImplIf((double)(12345), (long)(12345));
+			data_5 = new SomeIdentityClassImplIf((double)(12345), (long)(12345));
 
-         for (int i = 0; i < ARRAY_LENGTH; i++) {
-            data_1[i] = new SomePrimitiveClassImplIf((double)i, (long)i);
-            data_2[i] = new SomePrimitiveClassImplIf((double)(i+1), (long)(i+1));
+			for (int i = 0; i < ARRAY_LENGTH; i++) {
+				data_1[i] = new SomePrimitiveClassImplIf((double)i, (long)i);
+				data_2[i] = new SomePrimitiveClassImplIf((double)(i+1), (long)(i+1));
 
-            data_3[i] = data_5;
-            data_4[i] = new SomeIdentityClassImplIf((double)(i+1), (long)(i+1));
-         }
-      }
+				data_3[i] = data_5;
+				data_4[i] = new SomeIdentityClassImplIf((double)(i+1), (long)(i+1));
+			}
+		}
    }
 
    static void readArrayElementWithDoubleField(SomePrimitiveClassWithDoubleField[] data) throws Throwable {
```

Comment:
Again, thank you for cleaning up a lot of the lines that were indented with spaces. It looks like the static `{read|write}ArrayElementWith*` methods and their corresponding closing braces are still indented with spaces.

The corresponding lines in the lw5 version of the file are similarly affected.